server/sockets/helpers/index.js

```diff
@@ -44,43 +46,33 @@ const updateListOnDashboardAndCohortView = (
     .then(doc => {
       if (doc) {
         const { viewersIds, cohortId } = doc;
-        const dashboardClientExists = dashboardClients.size > 0;
-        const cohortViewClientsExists = cohortViewClients.size > 0;
 
-        if (dashboardClientExists) {
-          viewersIds.forEach(id => {
-            const viewerId = id.toString();
-            const list = responseWithList(doc, id);
+        viewersIds.forEach(id => {
+          const viewerId = id.toString();
+          const list = responseWithList(doc, id);
+
+          if (dashboardClients.has(viewerId)) {
+            const { socketId } = dashboardClients.get(viewerId);
+
+            socket.broadcast
+              .to(socketId)
+              .emit(ListActionTypes.FETCH_META_DATA_SUCCESS, {
+                [listId]: { ...list }
```

Comment:
if the list is object itself, do you have to spread here??
Or just ` [listId]: list `??